pages/Documents/ConversationalAI/Third-PartyBots/ibm-watson-assistant-version-2.md

```diff
@@ -50,24 +50,24 @@ You need to fill in the following infromation:
   </thead>
   <tbody>
   <tr>
-    <td>Workspace URL</td>
-    <td>Watson Assistant Workspace URL. Note that this workspace URL changes according to your account. In order to figure out which URL is correct for you, check the location of your Assistant in IBM's Cloud console. Then, choose the corresponding URL based on geo-location. For example, if your IBM location is "eu-gb", the London URL is the correct one for you.</td>
-    <td>https://api.eu-de.assistant.watson.cloud.ibm.com</td>
+    <td>Client ID</td>
```

Comment:
Sure that his should all go into ibm-watson-assistant-version-2.md ?

Reply:
no :) will fix it